<?xml version="1.0"?>
<!--
Launch the path follower with the pure pursuit controller for a bi-steerable robot.
-->
<launch>
    <group ns="path_follower">
        <group ns="controller">
            <!-- parameters, that are dependent of the controller or the robot model -->
            <group ns="2steer_purepursuit">
                <param name="k_forward" value="0.2" />
                <param name="k_backward" value="0.2" />
                <param name="vehicle_length" value="0.34" />
            </group>
        </group>
    </group>
</launch>
